client/src/document/organisms/sidebar/index.tsx

```diff
@@ -66,7 +66,6 @@ export function RenderSideBar({ doc }) {
       <SidebarContainer doc={doc}>
         {doc.sidebarHTML && (
           <>
-            <h4 className="sidebar-heading">Related Topics</h4>
```

Comment:
I'm a bit concerned what losing the heading means for a11y: we weren't using `aria-labelledby` correctly on the `<nav>`, but there was at least a heading as the first element. Now there's no indication of what the `<nav>` contains without tabbing through it: we should set an `aria-label` on it: https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-label

Reply:
Good point, fixed in 9d0c276acc36bc5b897d4c33a4b88ed2eac29fe1.